services/conservation_portfolios/src/util/types_data.ts

```diff
@@ -0,0 +1,6 @@
+export interface PortfoliosByTarget {
```

Comment:
I see why you changed the name of the other file, and it's something we have not discussed, but I think it's better one of these options:

- Create a folder  `src/types` and create files for each layer (example: `services.ts`, `routes.ts`, etc)
- Create a file `types.ts` inside each layer folder (example: `src/routes/types.ts`, `src/services/types.ts) (Unconsciously I think I implemented this)
- Create a folder  `src/types` and separate files by each service business model (exaple: `targets.ts`)

Or would be nice to know if any member has any other suggestion.

I don't think is a good idea to keep all types in the util folder, since types are not utilities. Also this separation of `_conf` and `_data` seems good now, but I thinks it's too general and may get out of control

Reply:
I chose 3er option. One file define types of microservice configuration and other for targets.

Reply:
I agree with that option, but I still think it's better `util` than config, since this types don't refer to some configuration but to cross-service utilities, alto this option group types by business model, util or utils fits better than config

Reply:
Done!